weblogic/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,36 @@
+name: WebLogic
+files:
+- name: weblogic.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/jmx
+  - template: instances
+    options:
+    - template: instances/jmx
+      overrides:
+        host.value.example: localhost
+        port.value.example: 9090
+  - template: logs
+    example:
+    - type: file
```

Comment:
This logs configuration is the same for admin servers and managed servers. Should I remove one so that it's not redundant?

Reply:
Perhaps we should suggest the service tag in that case? Because it wasn't clear to me what the service should be. This would also make it easier to filter logs if we have a suggested log example.

Reply:
i still see the service placeholder